uber/tests/conftest.py

```diff
@@ -50,13 +50,34 @@ def init_db(request, sensible_defaults):
             first_name='Regular',
             last_name='Attendee'
         ))
+
+        d_arcade = str(c.ARCADE)
+        d_console = str(c.CONSOLE)
+        d_arcade_and_console = '{},{}'.format(c.ARCADE, c.CONSOLE)
+        assigned_depts = {
+            'One': d_arcade,
+            'Two': d_console,
+            'Three': d_arcade_and_console,
+            'Four': d_arcade_and_console,
+            'Five': ''
+        }
+        trusted_depts = {
+            'One': '',
+            'Two': '',
+            'Three': '',
+            'Four': d_arcade_and_console,
+            'Five': ''
+        }
+
         for name in ['One', 'Two', 'Three', 'Four', 'Five']:
```

Comment:
new unit test stuff. instead of creating these assignments in the fixture, I assign them here when we actually create new rows in the database.